crypto/evp/keymgmt_lib.c

```diff
@@ -161,6 +157,48 @@ void evp_keymgmt_clear_pkey_cache(EVP_PKEY *pk)
                 keymgmt->freekey(provdata);
             EVP_KEYMGMT_free(keymgmt);
         }
+
+        pk->bits = 0;
```

Comment:
why do you only clear bits here and not the other 2 cached fields?

Reply:
'cause sloppy.  Fixed.  Thanks for catching it.